experimental/marginalization_mixup/batchensemble.py

```diff
@@ -723,6 +723,9 @@ def compute_acc_conf(preds, label, focus_class):
   final_checkpoint_name = checkpoint.save(
       os.path.join(FLAGS.output_dir, 'checkpoint'))
   logging.info('Saved last checkpoint to %s', final_checkpoint_name)
+  final_save_name = os.path.join(FLAGS.output_dir, 'model')
+  model.save(final_save_name)
```

Comment:
Are we going to remove checkpoint.save in the future and only use model.save?

Reply:
I think so. I'm adding model.save temporarily so I can easily load SavedModels from TS without having to re-build everything. Checkpoints are still needed for other workflows (e.g., Ghassen's).